src/wp-admin/includes/dashboard.php

```diff
@@ -1989,19 +1989,29 @@ function wp_dashboard_empty() {}
  * @since 3.3.0
```

Comment:
Let's add a `@since 5.9.0` here which describes how the functionality changed.

Reply:
Fixed in 2e0b4a1cfcc6a9139e39d5b1cf56438bc3451864